.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.5', '2.6', '2.7' ]
+        ruby: [ '2.5', '2.6', '2.7', '3.0', '3.1' ]
```

Comment:
Run the test suite against Ruby 3.0 and 3.1 in the CI build.

Reply:
Unfortunately, Ruby 3.2 doesn't pass. Some dependencies are not compatible with Ruby 3.2 it seems.